src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,298 @@
+            return "todo [task description]";
+        case "event":
+            return "event [event description] /at [date]";
+        case "deadline":
+            return "deadline [task description] /by [due date]";
+        case "done":
+            return "done [index of task]";
+        case "delete":
+            return "delete [index of task]";
+        default:
+            return null;
+        }
+    }
+    
+    public static void unknownCommand() {
+        System.out.println("\tSorry, this command is not available yet. Stay tuned! :)");
+    }
+    
+    public static void list() {
+        if (tasks.size()==0) {
+            System.out.println("\tYou have no tasks in your list");
+        } else {
+            System.out.println("\tHere are the tasks in your list:");
+            for (int i=0; i<tasks.size(); i++) {
+                Task currentTask = tasks.get(i);
+                System.out.println("\t" + Integer.toString(i+1) + ".[" + currentTask.getType() + "]["
+                        + currentTask.getStatusIcon() + "] " + currentTask.getDescription() + " " 
+                        + currentTask.printDate());
+            }
+        }
+    }
+
+    public static void addTodo(String command) {
+        String[] splitCommand = command.split(" ", 2);
+        if (splitCommand.length==1) {
+            System.out.println("\t!!! Please specify the task description !!!");
+            System.out.println("\tUsage: " + printUsage("todo"));
+        } else {
+            String todo = splitCommand[1];
+            Todo t = new Todo(todo);
+            tasks.add(t);
+            System.out.println("\tGot it. I've added this task:");
+            System.out.println("\t   [" + t.getType() + "][" + t.getStatusIcon() + "] " + t.getDescription());
+            if (tasks.size()==1) {
+                System.out.println("\tNow you have 1 task in the list.");
+            } else {
+                System.out.println("\tNow you have " + Integer.toString(tasks.size()) + " tasks in the list.");
+            }
+        }
+    }
+
+    public static void addEvent(String command) {
+        String[] splitCommand = command.split(" ", 2);
+        if (splitCommand.length==1) {
+            System.out.println("\t!!! Please specify event description !!!");
+            System.out.println("\tUsage: " + printUsage("event"));
+        } else {
+            try {
+                String description = splitCommand[1].split("/at")[0].strip();
+                String date = splitCommand[1].split("/at")[1].strip();
+                Event t = new Event(description, date);
+                tasks.add(t);
+                System.out.println("\tGot it. I've added this task:");
+                System.out.println("\t   [" + t.getType() + "][" + t.getStatusIcon() + "] "
+                        + t.getDescription() + " " + t.printDate());
+                System.out.println("\tNow you have " + Integer.toString(tasks.size()) + " tasks in the list.");
```

Comment:
Perhaps use SLAP and extract these statements out into methods? (there are a couple of instances of this in the code, so you can do the similar thing for all of those instances too)